test/unit/schema/oracledb.js

```diff
@@ -669,6 +669,16 @@ describe('OracleDb SchemaBuilder', function() {
     expect(tableSql[0].sql).to.equal(
       'alter table "users" add "foo" timestamp with local time zone'
     );
```

Comment:
Cool, there was a test case 👍

Reply:
Considering that there is a unit test for this, and output is same, do we really need to wait before merging?

Reply:
Yep, I don't believe that test work unless I can see that it has been ran successfully and I don't wan't to check that manually right now.